### PR TITLE
Recipe Model and Respository

### PR DESCRIPTION
**Pull request by armandojsalazar**
Made the recipe model and recipe repository.

### CODE REVIEW

File: src/main/java/com/example/bakingbuddy/demo/Model/Recipe.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(nullable = false, length = 100)
+    private String name;
+
+    @ManyToOne
+    private User owner;
+
+    @OneToMany(mappedBy = "owner")
```
Comment (asvarnon) [inline src/main/java/com/example/bakingbuddy/demo/Model/Recipe.java]: Change mappedBy to "recipe"

Comment (asvarnon) [inline src/main/java/com/example/bakingbuddy/demo/Model/Recipe.java]: start reviewing

**Review by asvarnon — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/bakingbuddy/demo/Model/Recipe.java (hunk 1 above)
Comment (armandojsalazar) [reply src/main/java/com/example/bakingbuddy/demo/Model/Recipe.java]: ok

**Review by armandojsalazar — COMMENTED** [review]
updated the file

**Pull request merged**